orangegrove.rb

```diff
@@ -0,0 +1,85 @@
+# ORANGE TREE ------------------------------------------------------------------------------->
+class OrangeTree
+
+	attr_reader :height, :age, :orange_count, :max_age, :dead
```

Comment:
Do you need to expose all of these variables? You have methods that return relevant information. Your `OrangeGrove` class uses the `age` and `orange_count` variables of your tree instance, but your other variables aren't used outside of your class. Try not to give outsiders more access than they need to your instance variables.